lib/diagnostic.js

```diff
@@ -15,7 +15,8 @@ db.dropDatabase();
 // House Targaryen, motto 'Fire and Blood'
 
 // your code begins here
-
+db.createCollection(houses);
+db.houses.insert({name: 'House Arryn', motto: 'As High as Honor'}).insert({name: 'House Stark', motto: 'Winter is Coming'}).insert({name: 'House Targaryen', motto: 'Fire and Blood'});
```

Comment:
These methods cannot be chained.  The documents must be added individually or all together as an array.

```javascript
db.houses.insert({
  name: 'House Arryn',
  motto: 'As High as Honor'
});

db.houses.insert({
  name: 'House Stark',
  motto: 'Winter is Coming'
});

db.houses.insert({
  name: 'House Targaryen',
  motto: 'Fire and Blood'
});
```

or

```javascript
db.houses.insert([
  {
    name: 'House Arryn',
    motto: 'As High as Honor'
  },
  {
    name: 'House Stark',
    motto: 'Winter is Coming'
  },
  {
    name: 'House Targaryen',
    motto: 'Fire and Blood'
  }
]);
```